product.json

```diff
@@ -28,5 +28,11 @@
 	"releaseNotesUrl": "https://go.microsoft.com/fwlink/?linkid=862039",
 	"documentationUrl": "https://go.microsoft.com/fwlink/?linkid=862277",
 	"commit": "9ca6200018fc206d67a47229f991901a8a453781",
-	"date": "2017-12-15T12:00:00.000Z"
+	"date": "2017-12-15T12:00:00.000Z",
+	"extensionsGallery": {
+		"serviceUrl":"https://raw.githubusercontent.com/Microsoft/sqlopsstudio/feature/extensionManager/extensionsGallery.json"
```

Comment:
We'll want to update this to use the master branch

Reply:
this is for testing. I'll change it later to the correct path

---
In reply to: [168322398](https://github.com/Microsoft/sqlopsstudio/pull/689#discussion_r168322398) [](ancestors = 168322398)